src/test/java/com/passbook/sparkeighteen/controller/UserControllerTest.java

```diff
@@ -96,4 +96,28 @@ public void validRequest_errorResponse_LoginSuccessfull() throws Exception {
         assertEquals(response.getStatusCode(), HttpStatus.OK);
 
     }
+
+    @Test
+    public void deleteProfile_validUserId_deleteProfileSuccessful() throws Exception {
+
+        Integer userID = 1;
+
+        Mockito.when(userService.deleteProfile(any())).thenReturn("user deleted " + userID);
+
+        ResponseEntity<String> response = userController.deleteProfile(userID);
+        assertEquals(response.getStatusCode(), HttpStatus.OK);
+
+    }
+
+    @Test
+    public void deleteProfile_UserIdNotExist_deleteProfileUnsuccessful() throws Exception {
```

Comment:
Again the name U is capital?
I can't keep pointing this out again and again

Reply:
updated